src/languageservice/parser/jsonParser07.ts

```diff
@@ -878,6 +882,30 @@ function validate(
     }
   }
 
+  function _validateDeprecated(node: ASTNode, schema: JSONSchema, validationResult: ValidationResult): void {
+    if (schema.deprecated) {
+      let location;
+      if (node.parent !== undefined && node.parent.type === 'property') {
+        location = {
+          offset: node.parent.offset,
+          length: node.parent.length,
+        };
+      } else {
+        location = { offset: node.offset, length: node.length };
+      }
+      validationResult.problems.push({
+        location,
+        severity: DiagnosticSeverity.Hint,
```

Comment:
Should this be a warning?

Reply:
Usually I would say the same deprecation as warning, but if they are doing hint for json language server, i would try to keep it the same, to avoid diverging.